### ISSUE TITLE
blurry slides with chalkboard+revealjs bug on safari 

### ISSUE THREAD

**Issue by cscheid** [quarto-dev/quarto-cli#10842]
Repro:

````
---
title: Chalkboard bug
format:
  revealjs:
    chalkboard: true
---
````

Open this document on Safari 16.6 (and presumably lower) and you see this:

<img width="1411" alt="image" src="https://github.com/user-attachments/assets/d7876f0f-4d19-4850-931c-e3b40e95c06b">

Notably, this is resolved if users upgrade to macOS Sequoia (!?)

**Comment by cscheid** [comment]
The bug happens because somehow we have a `.reveal > .overlay` element active on safari, which has the following style:

https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/revealjs/reveal/css/reveal.scss#L1446-L1456

The blur itself comes from upstream:

https://github.com/hakimel/reveal.js/blob/5.1.0/css/reveal.scss#L1446-L1456

**Comment by cscheid** [comment]
It's a chalkboard bug: https://github.com/rajgoel/reveal.js-plugins/issues/179

**Comment by cscheid** [comment]
I see https://github.com/rajgoel/reveal.js-plugins/pull/180, but it doesn't look like it really fixed the issue on Safari for us, @cderv

**Comment by cderv** [comment]
😞 It is a bad upstream bug - you found the right link. I used the patched version already 
[`dcd1724` (#10361)](https://github.com/quarto-dev/quarto-cli/pull/10361/commits/dcd172406aa38a600f7a17f7c0eb4313878b7033) - that is really annoying it does not work for safari (I also can't test it on windows I think... 🤔 )

Did you identify in CSS inspector why the rule is not applying ? 

If we can identify we can patch our version and report upstream.

**Comment by cscheid** [comment]
The rule at play is `.reveal > .overlay`, but if you look at it, that rule clearly has an intentional `blur()` filter. I suspect what's happening is that something in Safari is going wrong and the div should not be visible.

**Comment by cderv** [comment]
> The rule at play is .reveal > .overlay

This rule cover more usage than chalkboard, so I don't want to remove it. 

Following some hints about browser support here what I found: https://caniuse.com/?search=backdrop-filter

`backdrop-filter` CSS property is supported in Safari only recently since 18.0 from September 2024. So it would probably explain why "this is resolved if users upgrade to macOS Sequoia (!?)" as you said. 

For previous SAFARI, it seems we need `-webkit-backdrop-filter` to support the same

**Comment by cbrnr** [comment]
I reported a duplicate issue in #11297, but now I'm not sure if this was really the same problem. Is this fix already available in a pre-release? I just tried v1.6.32 and the text in my example is still blurry.

**Comment by cderv** [comment]
This is already in pre-release yes; But it was only happning when `chalkboard: true` as it was a patch required in chalkboard plugin itself. If have your issue without chalkboard, then this is not related.
